app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
The plan UX-wise was to add a permanent snackbar thingy with the number of pending observations and a "retry" button. Edit conflicts aren't a problem for observations because you can't edit them :)

cc @dancunningham